roles/rook/README.rst

```diff
@@ -0,0 +1,101 @@
+Rook
+========
```

Comment:
You should have an equal number of `=` characters as there are letters above it.

```
Rook
====
```

Reply:
Updated the readme file